src/main/java/green/fallingsand/Main.java

```diff
@@ -0,0 +1,27 @@
+package green.fallingsand;
+
+import java.io.IOException;
+
+public class Main {
+    public static int ENTER = 10;
+
+    public static void main(String[] args) {
+        Sand sand = new Sand(50, 10);
+        sand.randomSand(50);
+        System.out.println(sand.toString());
+        try {
+            while (System.in.read() == ENTER) {
```

Comment:
https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/awt/event/KeyEvent.html#VK_ENTER is better I think

Reply:
Make this change then I will approve it.